packages/core/src/formats.test.ts

```diff
@@ -3,50 +3,53 @@ import { date, number } from "./formats"
 describe("@lingui/core/formats", () => {
   describe("date", () => {
     it("should support Date as input", () => {
-      expect(date(["en"], new Date(2023, 2, 5))).toMatchInlineSnapshot(
-        `"3/5/2023"`
-      )
+      expect(date(["en"], new Date(2023, 2, 5))).toBe("3/5/2023")
```

Comment:
it might be intended to use `toMatchInlineSnapshot` to make sure `string` format is generated. But I will leave it up to @thekip

Reply:
if you compare `x` with `"3/5/2023"` using `toBe`, then it's immediately clear that `x` is a string, so I don't see that as an issue 🙂